apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -546,8 +591,14 @@ const NewPublication: FC<NewPublicationProps> = ({ publication }) => {
       ) : null}
       {showPollEditor ? <PollEditor /> : null}
       {showLiveVideoEditor ? <LivestreamEditor /> : null}
-      <OpenActions />
-      <LinkPreviews />
+      {Boolean(nftOpenActionEmbed) ? (
```

Comment:
how will swap OA will be rendered? I see only nft oa is checked here

Reply:
All this logic will be deffered to the component itself